2-mandatory/2-financial-times.js

```diff
@@ -6,15 +6,26 @@
 */
 function potentialHeadlines(allArticleTitles) {
     // TODO
+    const headTitle = allArticleTitles.filter(title => title.length < 65);
```

Comment:
This is a nice implementation!
I would ask you double-check one thing: the comment above says "65 characters or less". Could you update this code to meet that requirement?